modules/serverless-functions-adding-annotations.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-annotations.adoc
+
+[id="serverless-functions-adding-annotations_{context}"]
+= Adding annotations to a function
+
+.Procedure
+
+. Open the `func.yaml` file for your function.
+
+. For every annotation that you want to add, add the following YAML to the `annotations` section:
++
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+annotations:
+  color: "red"
+----
++
+Substitute `color: "red"` with your annotation.
```

Comment:
This would be much better if it's an example of a real annotation someone might use or if we just use generic variables, e.g.
`<annotation_name>: "<annotation_value>"`

Also, use callouts <1> like this to describe things in the code / tell the user to substitute things.